style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+.toolbar {
+  display: flex;
+  align-items: center;
+  margin-top: 70px;
+  margin-left: 26px;
+}
+
+.mylogo {
+  align-content: center;
+}
+
+.menu {
+  margin-left: 300px;
+}
+
+.headline {
+  margin-top: 40px;
+}
+
+#container-one {
+  display: flex;
+  flex-direction: row;
+}
+
+.first {
+  display: flex;
+  flex-direction: column;
+}
+
+.rect-6 {
+  width: 15px;
+}
+
+.ell-11 {
+  margin-top: -30px;
+}
+
+.second {
+  display: flex;
+  flex-direction: column;
+  margin-left: 100px;
+  margin-top: 70px;
+}
+
+.ell-5 {
+  width: 11px;
+  height: 10px;
+  margin-top: -10px;
+}
+
+.ell-8 {
+  margin-left: 130px;
+}
+
+.fourth {
+  display: flex;
+  flex-direction: column;
+  margin-left: 85px;
+}
+
+.ell-3 {
+  width: 60px;
+  margin-left: 50px;
+  margin-top: -45px;
+}
+
+.introduction {
+  font-size: 65px;
+}
+
+.profession {
+  font-size: 25px;
+  word-wrap: break-word;
+}
+
```

Comment:
- [ ] Your paragraph color looks different from Figma's design, kindly fix it.
![Screenshot from 2023-03-24 11-12-40](https://user-images.githubusercontent.com/91142432/227506565-48e5fa03-379b-45ff-ac7d-6735c35acdaa.png)